openapi.yaml

```diff
@@ -15104,6 +15043,65 @@ components:
         metadata:
           $ref: '#/components/schemas/video-status-encoding-metadata'
   parameters:
+    filterBy:
+      name: filterBy
+      description: |
+          Use this parameter to filter the API's response based on different data dimensions. You can serialize filters in your query to receive more detailed breakdowns of your analytics.
+
+          - If you do not set a value for `filterBy`, the API returns the full dataset for your project.
+          - The API only accepts the `mediaId` and `mediaType` filters when you call `/data/metrics/play/total`.
+          
+          These are the available breakdown dimensions:
+          
+          - `mediaId`: Returns analytics based on the unique identifiers of a video or a live stream.
+          - `mediaType`: Returns analytics based on the type of content. Possible values: `video` and `live-stream`. 
+          - `continent`: Returns analytics based on the viewers' continent. The list of supported continents names are based on the [GeoNames public database](https://www.geonames.org/countries/). You must use the ISO-3166 alpha2 format, for example `EU`. Possible values are: `AS`, `AF`, `NA`, `SA`, `AN`, `EU`, `AZ`. 
+          - `country`: Returns analytics based on the viewers' country. The list of supported country names are based on the [GeoNames public database](https://www.geonames.org/countries/). You must use the ISO-3166 alpha2 format, for example `FR`.
+          - `deviceType`: Returns analytics based on the type of device used by the viewers. Possible response values are: `computer`, `phone`, `tablet`, `tv`, `console`, `wearable`, `unknown`.
```

Comment:
Shouldn't we write the second sentence as "**Examples of** possible response values are", to show that list is non-exhaustive? I'm fairly certain I've seen "Desktop" for instance

Reply:
Good catch, you are right! Fixing ✅